<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connect to the VFD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>540</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="chartsTab">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Charts</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>500</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Here goes motor speed, torque (multiple y axes)</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="firstChartView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>500</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>481</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Here goes force, temperatur (multiple y axes)</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="secondChartView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>501</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="limitsTab">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Limits</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;#TODO:&lt;/p&gt;&lt;p&gt;Limit for speed, torque, temperature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="alarmsTab">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Alarms</string>
     </attribute>
     <widget class="QTextEdit" name="alarmsEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>541</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;#TODO:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Alarm history goes here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loggingTab">
     <attribute name="title">
      <string>Logging</string>
     </attribute>
     <widget class="QListView" name="dataDumpView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>241</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data dumps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log rate [Hz]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="logRateSpinbox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How often data are saved per second&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QListView" name="avaiableChannelsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaiable channels</string>
      </property>
     </widget>
     <widget class="QListView" name="loggingChannelsView">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>50</y>
        <width>231</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Logging channels</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toLoggingChannelsButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>23</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Move selected channels to logging channel</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toAvaiableChannelsButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>23</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createReportButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Create a single report file from the data dumps</string>
      </property>
      <property name="text">
       <string>Create report</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="testTab">
     <attribute name="title">
      <string>Testing</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Testing page for experiementell features</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="channelsTab">
     <attribute name="title">
      <string>Channels</string>
     </attribute>
     <widget class="QTableView" name="channelsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>291</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User can change channel's name</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>681</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="manualTab">
     <property name="toolTip">
      <string>Manual control mode</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="VFDEditor" name="vfdEditor" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="automationTab">
     <property name="toolTip">
      <string>Automatic control mode</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Automation</string>
     </attribute>
     <widget class="TestSequencer" name="testSequencer" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="loggingEnableButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enable logging</string>
    </property>
    <property name="text">
     <string>Logging Enable</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewConsole"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionNetworkSettings"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTestProfiletChecker"/>
   </widget>
   <widget class="QMenu" name="menuHelps">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUITour"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionVfdSetting"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
   <addaction name="menuView"/>
   <addaction name="menuOption"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelps"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVfdSetting">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>VFD Setting</string>
   </property>
   <property name="toolTip">
    <string>Setting for VFD</string>
   </property>
  </action>
  <action name="actionViewConsole">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/console.png</normaloff>:/images/console.png</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Messages between host PC and VFD</string>
   </property>
  </action>
  <action name="actionTestProfiletChecker">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/contra.png</normaloff>:/images/contra.png</iconset>
   </property>
   <property name="text">
    <string>Test Profile Checker</string>
   </property>
   <property name="toolTip">
    <string>Checker for test receipt</string>
   </property>
  </action>
  <action name="actionUITour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/tour.png</normaloff>:/images/tour.png</iconset>
   </property>
   <property name="text">
    <string>UI Tour</string>
   </property>
   <property name="toolTip">
    <string>A short guide how to navigate through UI</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/question_mark.png</normaloff>:/images/question_mark.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Show this application about page</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionNetworkSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VFDEditor</class>
   <extends>QWidget</extends>
   <header>vfdeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestSequencer</class>
   <extends>QWidget</extends>
   <header>testsequencer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
